### DISCUSSION TITLE
Please help

### DISCUSSION THREAD

**Discussion by AexaDe** [olikraus/u8g2#2254] [Q&A]
hello this is the first time i ask a question on Github and i dont know if i am doing it right.

Im facing a memory problem using uglib2 it uses to much memory (120%) of the available 2048bytes. when i remove the library from my sketch it occupies only 20%. Am i doing anything wrong? is there a way to make it more lightwheight i just need to print some text on a 128x64 oled screen i2c. I tried using adafruit's library but it has serious timing issues and crashes my Arduino.
Thank you in advance.

**Comment by olikraus** [comment]
See https://github.com/olikraus/u8g2/issues/2256
